types/common/buffer.d.ts

```diff
@@ -0,0 +1,232 @@
+declare namespace NodeJS {
```

Comment:
can we just "import" the node buffer definition instead of copying it here ?
(we anyway depend on node types)